content/hardware/04.pro/boards/portenta-x8/tutorials/image-flashing/content.md

```diff
@@ -0,0 +1,87 @@
+---
+beta: true
+title: 'How To Flash Your Portenta X8'
+description: 'This tutorial teaches you how to flash your Portenta X8 through USB'
+difficulty: intermediate
+tags:
+  - Embedded Linux
+  - Flashing
+  - Foundries.io
+author: 'Pablo Marquínez'
+hardware:
+  - hardware/04.pro/boards/portenta-x8
+---
+
+## Overview
+
+In this tutorial you will see how to manually flash your Portenta X8 with the image that is provided by Foundries.io through USB using the Terminal.
+
+## Goals
+
+- Get the required files
+- Set up the correct structure of the files
+- Set up the board
+- Flash the device
+
+### Required Hardware and Software
+
+- USB-C to USB-A or USB-C to USB-C
+- Portenta X8
+- Portenta Breakout Board <!-- or Portenta Max Carrier-->
+- Arduino Create account
+- Arduino Create Pro subscription
+- Foundries.io account (linked with the Pro plan)
+- FoundriesFactory® ([Check the Getting Started tutorial](https://docs.arduino.cc/tutorials/portenta-x8/out-of-the-box))
+- 1 Device already attached to your factory ([Check the Getting Started tutorial](https://docs.arduino.cc/tutorials/portenta-x8/out-of-the-box))
+    
+## Instructions
+
+### Get the Required Files
+
+Required files following this structure:
+
+- Root folder
+  - imx-boot
+  - imx-boot-portenta-x8
+  - sit-portenta-x8.bin
+  - u-boot-portenta-x8.itb
+  - mfgtool-files-portenta-x8.tar (Unzipped)
+  - lmp-partner-arduino-image-portenta-x8.wic.gz (Unzipped)
```

Comment:
@marqdevx is this the part that causes the linter error?

> `❌ ERROR: Content uses nested lists Location`

I like to use [the `tree -AF` command output](https://linux.die.net/man/1/tree) for file system diagrams:

```text
<Root folder>/
├── imx-boot
├── imx-boot-portenta-x8
├── lmp-partner-arduino-image-portenta-x8.wic.gz
├── mfgtool-files-portenta-x8.tar
├── sit-portenta-x8.bin
└── u-boot-portenta-x8.itb
```

Reply:
If im not wrong the issue is on the "meta-data" on top of the content

nice! I tried to do that manually but now i will use that 👀

Reply:
Not working in windows though :( ahha

Reply:
![image](https://user-images.githubusercontent.com/11246294/163394399-deec7e7a-0e6a-4d80-9fb9-accd45fb76b8.png)
Much better

Reply:
There is a native `tree /f` on Windows:
https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/tree
However, I don't really like the output.

I use this instead:
https://sourceforge.net/projects/gnuwin32/files/tree/
It will produce the same output as `tree` on Linux if you use the `-A` flag